.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   gdal-build-executor:
     docker:
-      - image: jamesmcclain/gdal-build-environment:4
+      - image: daunnc/gdal-build-environment:5
```

Comment:
Sorry @jamesmcclain, I had to publish it under my account.

Reply:
Passing of the torch